script.js

```diff
@@ -2,49 +2,73 @@
 //Declare a variable called 'name' that has the value of your name.
 
 /*code here*/
-
+var name="Jose";
 //#2
 //create an if statement that checks to see if your name is equal to 'Ben'
 //If yes, log 'yes'
 //otherwise log 'no'
 //note: use console.log to log the string
 
 /*code here*/
-
+if( name==='Ben')
+  Console.log("yes");
+else 
+  Console.log("no");
 //#3
 //create an array called 'favoriteFoods'.
 //fill it up with the names of several of your favorite foods
 
 /*code here*/
-
+var favoriteFoods=[
+  "Bananas","Apples","oranges","tika masala","pears", "California Burritos", "loquats"
+];
 //#4
 //use a for loop to log each food in the 'favoriteFoods' array
 
 /*code here*/
-
+for(var i=0;i<favoriteFoods.lenght;++i){
```

Comment:
I'd recommend spacing this out: `for (var i = 0; i < favoriteFoods.length; i++) {`.  Also, you usually see `i++` as a postfix operator instead of as a prefix.